utilities/must_gather.py

```diff
@@ -58,7 +58,7 @@ def run_must_gather(
     did_succeed, output, error = run_command(
         command=shlex.split(base_command),
         check=False,
-        timeout=TIMEOUT_20MIN,
+        timeout=TIMEOUT_60MIN,
```

Comment:
this affects all calls to `run_must_gather`
1. there's a bug in the existing code which does not use the passed arg `timeout`
2. please fix relevant calls to `timeout` and pass the required timeout.

Reply:
@rnetser thanks for your comment! As per my testing, the bug seems to be on the oc binary. The `--timeout` seems to be ignored.  The logs show that the param is correctly passed: 

```
subprocess.TimeoutExpired: Command '['oc', 'adm', 'must-gather', '--dest-dir=/data/tests-collected-info/must_gather_all_images', '--timeout=900s', '--all-images']' timed out after 1200 seconds
```
Are you aware of any bug in the oc binary? I took a look and didn't find any.

Reply:
Actually this value is not the one defined on `--timeout`, but the time that python will wait for the command to finish. This is the timeout that is taking place right now.

I've done some must-gather generations in a row and I observed times between 18 and 25 times. Therefore, I am updating this value from 20 minutes to 30 minutes, so the `oc adm must-gather` can finish before python terminates the command abruptly.